deployment/docker-compose.yml

```diff
@@ -78,6 +78,13 @@ uwsgi:
     - DJANGO_SETTINGS_MODULE=core.settings.prod_docker
     - VIRTUAL_HOST=healthsites.io
     - VIRTUAL_PORT=8080
+
+        # docker osm
+    - DOCKER_OSM_DATABASE_NAME=gis
```

Comment:
Didnt we discuss using another docker-compose entry for docker-osm here so that it runs in the same orchestration as hs.io? Then you can just reference the host by name...

Reply:
hi @timlinux 
already fixed it